src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -225,9 +166,30 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
     wallet,
   ])
 
-  const [serializedTxIndex, setSerializedTxIndex] = useState<string>('')
+  const thorchainNotationAssetId = useMemo(() => {
+    // TODO(gomes): rename the utils to use the same terminology as well instead of the current poolAssetId one.
+    // Left as-is for this PR to avoid a bigly diff
+    return assetIdToPoolAssetId({ assetId: poolAssetId })
+  }, [poolAssetId])
 
-  const tx = useAppSelector(state => selectTxById(state, serializedTxIndex))
+  const memo = useMemo(() => {
+    if (thorchainNotationAssetId === undefined) return
+    if (opportunityType === 'sym' && !pairAssetAddress) return
+
+    return isDeposit
+      ? `+:${thorchainNotationAssetId}:${pairAssetAddress ?? ''}:ss:${confirmedQuote.feeBps}`
+      : `-:${thorchainNotationAssetId}:${confirmedQuote.withdrawalBps}`
```

Comment:
These strings are important enough to abstract into helpers for re-use around lp generally

Reply:
Coming to theaters near you soon: super safe memos, fully tested: https://github.com/shapeshift/web/pull/6755 first with ensuring we get the affiliate name everywhere, and up next, with more building capabilities to *compose* them